pset-functions.js

```diff
@@ -27,10 +27,16 @@ Use the following test cases to confirm your program meets the success criteria
    Result: 21
 
 ************/
-console.log('Problem 1:')
+
 
 // Add your code below this line
 
+let blackJack = [22,22]
```

Comment:
@jimernest90 `blackjack` was meant to be a function that returns a single value. Take a look at the one such implementation approach found [here](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-functions.js#L33)